language/move-analyzer/editors/code/src/configuration.ts

```diff
@@ -25,17 +26,19 @@ export class Configuration {
 
     /** The path to the move-analyzer executable. */
     get serverPath(): string {
-        const defaultName = 'move-analyzer';
-        const path = this.configuration.get<string>('server.path', defaultName);
-        if (path.length === 0) {
+        const defaultName = '~/.cargo/bin/move-analyzer';
```

Comment:
Are these changes strictly necessary for the new functionality to work? Perhaps they should become part of another PR to make understanding the current code changes easier?

Reply:
The previous default value configuration, even if I configure the PATH, it will prompt that the executable program cannot be found [here](https://github.com/move-language/move/blob/548f632ca3ace2707c59c96dda198a9a7bead592/language/move-analyzer/editors/code/src/context.ts#L22).

Reply:
And it's a useful change, but should be part of a separate small PR as it seems independent on what the current PR aims to support.

Reply:
Got it, I create a new PR

Reply:
New [PR](https://github.com/move-language/move/pull/224) is ok.